crates/wast/src/component/types.rs

```diff
@@ -317,6 +317,7 @@ impl Peek for PrimitiveValType {
                 | Some(("float64", _))
                 | Some(("char", _))
                 | Some(("string", _))
+                | Some(("error-context", _))
```

Comment:
I'll note that this was an accidental omission from https://github.com/bytecodealliance/wasm-tools/pull/2060